lms/static/sass/course/_info.scss

```diff
@@ -58,58 +58,6 @@ div.info-wrapper {
     width: 100%;
     display: block;
 
-    div.dates-banner {
```

Comment:
This block was removed, because it doesn't seem to be used anywhere? I found it because it shared a name with some other sass I was removing, but this is unrelated and unused.